src/components/TabPanel/index.tsx

```diff
@@ -79,18 +85,55 @@ export class TabPanel extends React.Component<TabPanelProps> {
         const navCx = 'cr-tab-panel__navigation-container-navigation';
 
         return (
-            <div className={className}>
-                <div className="cr-tab-panel__navigation-container draggable-container">
-                    <div className={navCx} role="tablist">
-                        {panels.map(this.renderTabPanel)}
+            <React.Fragment>
+                {!dropdown ? (
+                    <div className={className}>
+                        <div className="cr-tab-panel__navigation-container draggable-container">
+                            <div className={navCx} role="tablist">
+                                {panels.map(this.renderTabPanel)}
+                            </div>
+                            {optionalHead && <div className="cr-tab-panel__optinal-head">{optionalHead}</div>}
+                        </div>
+                        {contents}
                     </div>
-                    {optionalHead && <div className="cr-tab-panel__optinal-head">{optionalHead}</div>}
-                </div>
-                {contents}
-            </div>
+                ) : (
+                    <div className={className}>
+                        <div className="cr-tab-panel__navigation-container draggable-container">
+                            <div className="cr-tab-panel__dropdown">
+                                <DropdownComponent list={this.dropdownLabels()} className="cr-tab-panel__dropdown" onSelect={this.handleOrderTypeChange} placeholder=""/>
```

Comment:
List must use transltions

Reply:
But list always use panels.label values. And panels have translations here - https://github.com/openware/baseapp/blob/feature/trading-page-dropdown/src/mobile/components/Orders/index.tsx#L70